app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -71,17 +71,17 @@ $ske-course-video$
 
 They're usually online, but some courses can be classroom based and some can be a mixture of both. 
 
-Course lengths usually range from 8 to 28 weeks and are 25 hours per week. Your provider will decide which course length is appropriate based on your gap in knowledge. Different providers may offer different course lengths.
+Course lengths usually range from 8 to 28 weeks and are 25 hours per week. Your provider will decide which course length you need based on your gap in knowledge. Different providers may offer different course lengths.
```

Comment:
```suggestion
Course lengths range from 8 to 28 weeks and are 25 hours per week. Your provider will decide which course length you need based on your gap in knowledge. Different providers may offer different course lengths.
```
SKE team suggested removing `usually` - makes sense if all courses are between 8 and 28 weeks?